docs/app/tooling/IDE-integration.mdx

```diff
@@ -73,6 +73,8 @@ with Cypress.
 - [Test Utils](https://marketplace.visualstudio.com/items?itemName=chrisbreiding.test-utils):
   Easily add or remove `.only` and `.skip` modifiers with keyboard shortcuts or
   the command palette.
+- [Cypress Test Explorer](https://marketplace.visualstudio.com/items?itemName=dpanshug.cypress-test-explorer): 
```

Comment:
```suggestion
- [Cypress Test Explorer](https://marketplace.visualstudio.com/items?itemName=dpanshug.cypress-test-explorer):
```

Remove trailing space, which otherwise causes Prettier error